### DISCUSSION TITLE
I made you a discord server

### DISCUSSION THREAD

**Discussion by TheTwoNamedMan** [naikari/naikari#19] [General]
I have loved these escape velocity like games for a long while, and I noticed that your only way to communicate is through github and email, so I decided to make a discord server(if you don't want it I can delete it)
here is the link
https://discord.gg/HqKuSmnQ

**Comment by onpon4** [comment]
I don't mind. The only reason we haven't created a Discord server ourselves is because we haven't noticed enough interest to warrant one. But if you wanna run a server, that's fine with us. I should note, thô: I and others can't get into it with that link. You'll need to post an invite link.

🦇

**Reply by TheTwoNamedMan** [reply]
oh, I'll fix that
